src/server/utils/storyblok.ts

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios'
```

Comment:
Not a big thing but do we need `axios` here? I'm not exactly against it but I'm not seeing anything that we couldn't easily accomplish with `fetch` 🙂

Reply:
well it happens server-side so we need a library for it. I personally i like axios more than `node-fetch`, but i can switch it out if you want me to? :)

Reply:
No, I think this is all good 🙂We may want to switch on `giraffe` as well given that it uses `node-fetch` 🙂

Reply:
hm, maybe. axios' typings are quite awesome right? Unsure how it compares to `node-fetch` though?

Reply:
Haha 😄